subgraph/templates/subgraph.template.yaml

```diff
@@ -29,8 +29,7 @@ dataSources:
         - name: Poll
           file: ./node_modules/maci-contracts/build/artifacts/contracts/Poll.sol/Poll.json
       eventHandlers:
-        - event: DeployPoll(uint256,indexed uint256,indexed
-            uint256,(address,address,address))
+        - event: DeployPoll(uint256,indexed uint256,indexed uint256,(address,address,address),uint8)
```

Comment:
Don't forget to update tests with new event params. I think it's the reason why the subgraph task is failed.